spec/consensus/proposer-based-timestamp/tla/runApalache.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
```

Comment:
please set errexit and pipefile in all shell scripts.

Reply:
Feel welcome to propose the change here.

Reply:
add this line:

```sh
set -euo pipefail
```